src/PowerShellEditorServices/Language/AstOperations.cs

```diff
@@ -337,5 +326,28 @@ static public string[] FindDotSourcedIncludes(Ast scriptAst)
 
             return dotSourcedVisitor.DotSourcedFiles.ToArray();
         }
+
+        private static MethodInfo GetExtentCloneMethod(Ast scriptAst)
```

Comment:
I think this would be better as just a static field.  If your looking for how you could do that without an instance, you could do this:

```csharp
private static MethodInfo s_extentCloneWithNewOffset =
    typeof(PSObject).Assembly.GetType("System.Management.Automation.InternalScriptExtent")
        .GetMethod(...
```

Maybe set in a static constructor because of the compiler directives.

Reply:
Yeah I was wondering what the best way to do this was. Figured we'd find a better way in the PR. Thanks!